optimum/utils/input_generators.py

```diff
@@ -575,3 +575,72 @@ def generate(self, input_name: str, framework: str = "pt"):
                 max_value=1,
                 framework=framework,
             )
+
+
+class DummyStableDiffusionInputGenerator(DummyInputGenerator):
```

Comment:
I think this should be broken down in parts for the proper modalities.

- `input_ids` are handled already somewhere else.
- `encoder_hidden_states` are handled in `DummySeq2SeqDecoderTextInputGenerator` as `encoder_outputs`
- `sample` and `latent_sample` could come from a `DummyNoiseInputGenerator` or something of the likes
- I do not know where `timestep` should be handled because I do not know its purpose, but we could do something similar.